### PR TITLE
task/API-961 Create Coverage Experience API

### PR DESCRIPTION
**Pull request by esubocz-va**
Create Coverage experience API
Implement swagger
Add tests

Related Story: https://vasdvp.atlassian.net/browse/API-961

### CODE REVIEW

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    description = "Record found",
+    content =
+        @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = Coverage.Bundle.class)
+        )
+  )
+  @ApiResponse(
+    responseCode = "400",
+    description = "Not found",
```
Comment (dtournour-va) [inline urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: These descriptions are backwards. 400 is bad request

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    content =
+        @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = OperationOutcome.class)
+        )
+  )
+  @ApiResponse(
+    responseCode = "404",
+    description = "Bad request",
```
Comment (dtournour-va) [inline urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: 404 is not found

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      description = "Record found",
+      content =
+      @Content(
+          mediaType = "application/fhir+json",
+          schema = @Schema(implementation = Coverage.class)
+      )
+  )
+  @ApiResponse(
+      responseCode = "400",
+      description = "Not found",
```
Comment (dtournour-va) [inline urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: same

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      content =
+      @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = OperationOutcome.class)
+      )
+  )
+  @ApiResponse(
+      responseCode = "404",
+      description = "Bad request",
```
Comment (dtournour-va) [inline urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: same

Comment (esubocz-va) [reply urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: 👍

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java (hunk 3 above)
Comment (esubocz-va) [reply urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: 👍

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java (hunk 2 above)
Comment (esubocz-va) [reply urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: 👍

File: urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java (hunk 1 above)
Comment (esubocz-va) [reply urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java]: 👍

**Pull request merged**